pcmdi_metrics/precip_distribution/README.md

```diff
@@ -0,0 +1,23 @@
+# Precip distribution metrics
```

Comment:
Would it be possible to add a little information to the README about what types of input data the driver takes? For example is it daily timeseries, monthly averages, etc, and which variables. Also do you have to run the driver separately for each dataset, or can you run multiple datasets at once?

Reply:
Thank you for your suggestions. I have added the information to the README.